sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-full-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -196,6 +196,24 @@ describe(
       // TODO Add other fields as needed.
     });
 
+    it("Should generate disbursement file with formatted records when there is no data for the ecert generation.", async () => {
```

Comment:
`Should generate an e-cert file with only header and footer when there is no disbursement to be sent.` ?